src/lib/holocene/combobox/combobox.svelte

```diff
@@ -415,7 +418,15 @@
         type="text"
         value={displayValue}
         class:disabled
-        class={merge('combobox-input', className)}
+        class={merge(
+          'combobox-input',
+          multiselect
```

Comment:
> Could we give a wee bit more space after the leading icon?

@Alex-Tideman made some updates to our `Combobox` component (separate from the `ChipInput`) ⬇️ 

![Screenshot 2025-03-27 at 9 23 43 AM](https://github.com/user-attachments/assets/45de11ea-1758-4fc8-be42-5b362d87c50f)

Reply:
🤦 I didn't realize that was Combobox. What's the different between the combobox with chips and chip input?

Reply:
Ok just input and tab to create Chip, where Combobox is pick from select to create Chip

Reply:
Yeah we might want to revisit some of our inputs... I think we can narrow them down.